.github/workflows/transport-dependency-update.yml

```
@@ -0,0 +1,114 @@
+name: Transport Dependency Update
+
+on:
+  push:
+    tags:
+      - "core/v*" # Triggers on tags that start with core/v
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)